src/main/java/org/carlmontrobotics/subsystems/Drivetrain.java

```diff
@@ -330,9 +352,9 @@ public void initSendable(SendableBuilder builder) {
         builder.addBooleanProperty("Gyro Calibrating", gyro::isCalibrating, null);
         builder.addBooleanProperty("Field Oriented", () -> fieldOriented,
         fieldOriented -> this.fieldOriented = fieldOriented);
-        builder.addDoubleProperty("Odometry X", () -> getPose().getX(), null);
-        builder.addDoubleProperty("Odometry Y", () -> getPose().getY(), null);
-        builder.addDoubleProperty("Odometry Heading", () ->
+        builder.addDoubleProperty("Pose Estimator set x", () -> getPose().getX(), null);
+        builder.addDoubleProperty("Pose Estimator set y", () -> getPose().getY(), null);
+        builder.addDoubleProperty("Pose Estimator set rotation", () ->
```

Comment:
if you want to make the properties "settable" you need to provide a setter function instead of `null`. IMO your smartdashboard stuff is already enough for testing purposes so you don't have to do that, but naming it "set x" makes people think you can set the value when in reality you actually can't.